source/common/upstream/subset_lb.cc

```diff
@@ -38,9 +39,14 @@ SubsetLoadBalancer::SubsetLoadBalancer(
       subset_selectors_(subsets.subsetSelectors()), original_priority_set_(priority_set),
       original_local_priority_set_(local_priority_set),
       locality_weight_aware_(subsets.localityWeightAware()),
-      scale_locality_weight_(subsets.scaleLocalityWeight()), list_as_any_(subsets.listAsAny()) {
+      scale_locality_weight_(subsets.scaleLocalityWeight()), list_as_any_(subsets.listAsAny()),
+      time_source_(time_source) {
   ASSERT(subsets.isEnabled());
 
+  if (common_config.has_slow_start_config()) {
```

Comment:
Subset + Roundrobin /LeastRequest is supported.

Reply:
fixed